workers/social/lib/social/models/gitprovider/constants.coffee

```diff
@@ -2,5 +2,5 @@ module.exports =
   GITHUB_HOST     : 'github.com'
   RAW_GITHUB_HOST : 'raw.githubusercontent.com'
   GITLAB_HOST     : 'gitlab.com'
-  TEMPLATE_PATH   : 'Koding.yaml'
-  README_PATH     : 'Koding.README'
+  TEMPLATE_PATH   : '.koding.yml'
```

Comment:
right, we can add multiple template file supports which I didn't want to do that right now since it causes n times more api calls for getting the file content. but it's doable of course.